app/digests/api.py

```diff
@@ -1,26 +1,54 @@
+import json
+import os
 from typing import Any, Dict
 
 from django.conf import settings
-from rest_framework import viewsets
+from elife_api_validator import SCHEMA_DIRECTORY
+from jsonschema import validate as validate_json
+from jsonschema import ValidationError
+from rest_framework import viewsets, status
+from rest_framework.request import Request
 from rest_framework.response import Response
 
+from digests.exceptions import generate_error_string
 from digests.models import Digest
 from digests.pagination import DigestPagination
 from digests.serializers import DigestSerializer
 
 
+def get_schema(schema_name: str) -> Dict:
+    with open(os.path.join(SCHEMA_DIRECTORY, schema_name)) as schema:
+        val = json.loads(schema.read())
+        return val
+
+
 class DigestViewSet(viewsets.ModelViewSet):
     model = Digest
     queryset = Digest.objects.all()
     serializer_class = DigestSerializer
     pagination_class = DigestPagination
 
     content_type = settings.DIGESTS_CONTENT_TYPE
+    schema_name = 'digest.v1.json'
```

Comment:
The `Content-Type` header on the request should be matched to this, rather than it being hard-coded.